Dalamud/Utility/Signatures/SignatureHelper.cs

```diff
@@ -24,7 +26,8 @@ public static class SignatureHelper
     /// </summary>
     /// <param name="self">The object to initialise.</param>
     /// <param name="log">If warnings should be logged using <see cref="PluginLog"/>.</param>
-    public static void Initialise(object self, bool log = true)
+    /// <returns>Collection of created IDalamudHooks.</returns>
+    internal static IEnumerable<IDalamudHook> Initialise(object self, bool log = true)
```

Comment:
Consider renaming to `Initialize` to match US EN as used in the rest of the project.